### PR TITLE
cli/tx_cookbook_test

### PR DESCRIPTION
**Pull request by tantv4668**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/pylons/client/cli/tx_cookbook_test.go [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+			desc:       desc,
+			developer:  dev,
+			version:    newVersion,
+			email:      email,
+			enabled:    enabled,
+			common:     util.CommonArgs(acc[0].String(), net),
+			shouldFail: true,
+		},
+		{
+			testDesc:   "Name is to short",
```
Comment (mustafapylons) [inline x/pylons/client/cli/tx_cookbook_test.go:256]: "Name is too short", misspelled.

File: x/pylons/client/cli/tx_cookbook_test.go [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+			desc:       desc,
+			developer:  dev,
+			version:    newVersion,
+			email:      email,
+			enabled:    enabled,
+			common:     util.CommonArgs(acc[0].String(), net),
+			shouldFail: true,
+		},
+		{
+			testDesc:   "Description is to short",
```
Comment (mustafapylons) [inline x/pylons/client/cli/tx_cookbook_test.go:268]: "Description is too short" misspelled.

**Review by mustafapylons — COMMENTED** [review]
Some typos, otherwise it's good.

**Comment by MikeSofaer** [comment]
It would be good to remove the limits on name and description

File: x/pylons/client/cli/tx_cookbook_test.go (hunk 1 above)
Comment (tantv4668) [reply x/pylons/client/cli/tx_cookbook_test.go:256]: yah, thanks

**Review by faddat — APPROVED** [review]

**Pull request closed without merging**